src/state.ts

```diff
@@ -415,12 +415,13 @@ export class FormState<
   }
 
   @action
-  async processAll() {
+  async processAll(liveOnly?: boolean) {
     if (this.processor == null) {
       throw new Error("Cannot process all without backend configuration");
     }
+    this.setSaveStatus("after");
```

Comment:
Is this still needed?

Reply:
If you don't send a parameter by hand, the validations will look at the saveState. If we do not pass a parameter, and we don't set saveState beyond "before", we have the same issue as before and warnings are not triggered

Reply:
I do think removing this line and falling back on the save status would be ok. At that point we can rely on our previous system of only executing live validations after a save, unless we specifically specify otherwise with the given override.

If we keep this line we basically remove the "save status" functionality from `processAll`.

Reply:
Okay, but then then default behaviour remains the same and we have to make sure to explicitly pass the parameter in our application